ngraph/python/src/pyngraph/ops/constant.cpp

```diff
@@ -90,6 +90,18 @@ void regclass_pyngraph_op_Constant(py::module m)
                           const std::vector<uint64_t>&>());
 
     constant.def("get_value_strings", &ngraph::op::Constant::get_value_strings);
+    constant.def("get_vector", &ngraph::op::Constant::get_vector<char>);
```

Comment:
@ilyachur from my point of view get_vector method is useless because it makes a copy and fails if given type mismatch Constant type and can be replaced with cast_vector

Reply:
Agree.